libsolidity/ast/ASTJsonConverter.h

```diff
@@ -133,6 +133,7 @@ class ASTJsonConverter: public ASTConstVisitor
 	std::string location(VariableDeclaration::Location _location);
 	std::string contractKind(ContractDefinition::ContractKind _kind);
 	std::string functionCallKind(FunctionCallKind _kind);
+	std::string tokenKind(Token::Value _token);
```

Comment:
tokenKind() is only for literaltokens....therefore i renamed it to literalTokenKind()